text/0061-core-builder.md

```diff
@@ -4,9 +4,9 @@
 
 Over the past, the Paketo project has created four different stacks and builders based on the Ubuntu Linux distribution. The difference is in the number of packages included which in turn is based on the requirements of the different buildpacks included.
 This RFC proposes to create a new stack repository that will create a single `build` image and a number of `run` images based on the Ubuntu distribution and replace the existing `static`, `tiny` and `base` stacks. The `full` stack will be deprecated immediately, making the Jammy stack the last iteration of it.
-This RFC further proposes to create a new builder repository that will create a single builder using the build image as a base layer and the `base` run image as the default `run` image.
+This RFC further proposes to create a new builder repository that will create a two builders, both using the build image as a base layer and the `base` run image as the default `run` image. One builder will not contain any buildpacks and the other will contain a full set of Paketo buildpacks.
```

Comment:
```suggestion
This RFC further proposes to create a new builder repository that will create a two builders, both using the build image as a base layer and the `base` run image as the default `run` image. One builder will not contain any buildpacks and the other will contain Paketo buildpacks.
```

Yes, probably too much to get into here. Just want to say that one is buildpackless and one has buildpacks. I don't think we need further detail than that here.